src/runtimes/node/utils/entry_file.ts

```diff
@@ -8,11 +9,21 @@ export interface EntryFile {
   filename: string
 }
 
-const getEntryFileContents = (mainPath: string, moduleFormat: string) => {
+const getEntryFileContents = (mainPath: string, moduleFormat: string, layers: string[]) => {
   const importPath = `.${mainPath.startsWith('/') ? mainPath : `/${mainPath}`}`
 
   if (moduleFormat === ModuleFormat.COMMONJS) {
```

Comment:
We'll need to implement a similar bootstrap code for ESM functions. We're not generating any of those at the moment, so we can do that in a follow up PR.